### ISSUE TITLE
RAM vs. GoogLeNet_cars on custom dataset

### ISSUE THREAD

**Issue by joeysu** [liu-xb/RAM#1]
Hi, first of all, thanks for your great work on Vehicle-ReID.

Recently I run your fine-tuned model on my own dataset which are front car images similar to Veri.

I used pycaffe to extact features from 7 fc layers
**[att-branch-fc, fc7, fc7-bn, fc-fc-part-1, fc-fc-part-2, fc-fc-part-3, concat-features-parts]** 
and concatenate them for retrieval.

But I found that top-10 accuracy of RAM is lower than [GoogLeNet_cars](https://github.com/BVLC/caffe/wiki/Model-Zoo#googlenet_cars-on-car-model-classification).

RAM: 55.3%
GoogLeNet_cars: 78.1%

Have I missed something?

**Comment by MOONBEAR0703** [comment]
@s5plus1 hi. can you download the GoogLeNet_cars pretrained model?

**Comment by joeysu** [comment]
@MOONBEAR0703 Yes, I can.

**Comment by MOONBEAR0703** [comment]
@s5plus1 can you provide a link to me? Thanks. I cannot download it.

**Comment by joeysu** [comment]
@MOONBEAR0703 http://mmlab.ie.cuhk.edu.hk/datasets/comp_cars/googlenet_finetune_web_car_iter_10000.caffemodel

**Comment by liu-xb** [comment]
@s5plus1  Sorry for the late response! I just notice this issue.
The low performance might be mainly caused by the shallow backbone network. Our model is based on VGG_CNN_M, which is rather shallower than GoogLeNet. [This paper](http://www.eecs.qmul.ac.uk/~xiatian/papers/Kanac%C4%B1EtAl_BMVC2017WS.pdf) shows that a deep model (InceptionV3) trained with vehicle attribute classification task only and without ID annotation could achieves a good performance.
Another reason might be that the dataset we trained our model on is not generalized to other datasets well. VeRi dataset  has only 9 vehicle types, 10 colors and 576 vehicle IDs. While the dataset used to train GoogLeNet_cars has 431 car models, which might let the model generalize to other dataset better than VeRi.

**Comment by joeysu** [comment]
@liu-xb That makes sense. Thanks!

**Comment by liu-xb** [comment]
@s5plus1 My pleasure. Please feel free to let me know if you have any question.
